packages/twenty-website/src/content/user-guide/functions/workflows.mdx

```diff
@@ -0,0 +1,151 @@
+
+### Send Emails
+
+This action allows you to send an email from your workflow. To use this action:
+
+1. Add an email account in **Settings > Accounts**
+2. Enter the recipient’s address, subject, and message body to compose your email
+
+You can also reference variables from outputs of previous steps to further customize an action. Soon, you will have the capability to view attachments and format email content. 
+
+### Code 
+
+This action runs custom Javascript within your workflow. You can use this action to handle logic that goes beyond built-in actions such as transforming data, performing calculations, or integrating with external APIs. 
+
+Any variables you create and return here can be reused in later steps of the workflow. You can also test your code directly in this step and view the output to confirm it works as expected.
+
+Note: You can manage keys and access your API Playground [here](https://app.twenty.com/settings) or navigate to Settings > API & Webhooks (under Developers).
+
+### Form 
+
+This action allows you to prompt a form during the workflow to collect input from the user. The data collected can be used in later steps. To add this action, configure the input fields by selecting the type, label, and placeholder for each. 
+
+### HTTP Request
+
+This action allows you to send a request to an external API as part of your workflow; it can be used to integrate with external services, send or receive data, or trigger third-party actions. To use this action:
+
+1. Paste the endpoint you want to call
+2. Select an HTTP method from `GET`, `POST`, `PUT`, `PATCH`, or `DELETE`
+3. Enter any required header inputs and their values 
+4. For `POST`, `PUT`, or `PATCH`  include the body input to send with the request
+5. Paste a sample response body to preview its structure and reference keys in later steps
+
+## Executing Workflows
+
+Workflows can be executed in three ways: **Activate**, **Run**, and **Test**. Each serves a different purpose depending on where you are in the workflow setup.
+
+### Activate 
+
+To activate your workflow, click **Activate** at the top to run your workflow in the background. Each activation creates a new version and moves the workflow out of draft mode.
+
+### Runs
+
+Runs are saved executions of active workflows. They provide details like status, output, author, and last updates. View past runs from the **Runs** field or open the **Workflow Runs** view for more specific insights that’ll be useful for monitoring and troubleshooting.
+
+### Test
+
+You can test workflows before activating them. This is ideal for trigger types like **Launch Manually (no record selected)** or for testing specific action nodes like **Code**. Note: Testing a workflow or individual node does not activate it. The workflow remains in draft mode.
+
+## Workflows Statuses
+
+Workflows have three statuses: **Draft**, **Activated**, and **Deactivated**.
```

Comment:
The statuses are:

- `Draft`
- `Active`
- `Deactivated`
- `Archived`

The draft version is the current version being edited and has not been published yet.
The active version was a _deactivated_ or _draft_ version that was subsequently activated.
The deactivated version was an _active_ version that was subsequently deactivated.

All previous versions are archived, except for the most recent one. If I have: v1, v2 (active), and v3 (draft), v1 is necessarily archived. If I activate v3, v2 will also be archived.

You can't directly activate an archived version; you must _use it as a draft_.